src/main/resources/assets/admin/common/js/ui/treegrid/TreeGrid.ts

```diff
@@ -1898,7 +1901,7 @@ export class TreeGrid<DATA extends IDentifiable>
         return nodes.map((node: TreeNode<DATA>) => node.getData());
     }
 
-    highlightItemById(dataId: string, expand: boolean = false) {
+    highlightItemById(dataId: string, notify: boolean = true, expand: boolean = false) {
```

Comment:
If you add a new argument to an existing public method you MUST make it last, otherwise it might break calls of this method.

Reply:
Yep, I'm aware of this, but this is a new method Sasha implemented, so I believe no other place uses it so far...